src/js/events/MarathonActions.js

```diff
@@ -133,9 +150,28 @@ var MarathonActions = {
     });
   },
 
-  deleteService(serviceId) {
+  /**
+   * Delete a service (app, framework, or pod)
+   *
+   * @param {Service} service - the service you want to delete
+   */
+  deleteService(service) {
+    if (!(service instanceof Service)) {
+      if (process.env.NODE_ENV !== 'production') {
+        throw new TypeError('service is not an instance of Service');
+      }
+
+      return;
+    }
```

Comment:
I like the defensiveness here. I wonder whether we should adopt this as a pattern in general?